src/oaklib/converters/obo_graph_to_fhir_converter.py

```diff
@@ -290,3 +298,70 @@ def _convert_meta(self, source: Node, concept: Concept):
                     value=synonym.val,
                 )
             )
+
+
+@dataclass
+class OboGraphToFhirNpmConverter(OboGraphToFhirJsonConverter):
+    """Converts an OBO Graph to a FHIR NPM package.
+
+    Plays the same role as OboGraphToFhirJsonConverter, but also packages the outpus.
+    """
+
+    def dump(
```

Comment:
@cmungall I followed your suggestion and made a `fhirnpm` `output_type`. Given the architecture of OAK, I think a `OboGraphToFhirNpmConverter ` is expected. And it seemed to make sense to subclass it from `OboGraphToFhirJsonConverter`.

## Non-standard `dump()` and alternative idea
All it needs to do is override `dump()` and do some extra things. But this might violate your intentions of standardization around the `dump()` methods in these classes. Is it OK how I've done this? Or do you want me to create some kind of intermediate method, say, `package()` which comes after the inherited `OboGraphToFhirJsonConverter.convert()` and then returns back to `dump()`? If so, perhaps `package()` could do everything up to creating the actual zip file.

## Required `dump()` params
Some other non-standard changes that I've made to `OboGraphToFhirNpmConverter.dump()` given its special case:
- `target` is now a required param, and in this case represents the output directory. I could change this to be the output path of the zip file. However, it seems that it must be required, because the other methods print the output to the terminal, and I imagine we don't want to do that with a zip file.
- `manifest_path` a required parameter. If you like my idea of creating a `package()` method, I could pass this down and require it there instead.
- `kwargs["code_system_id"]`: I didn't make this a required param (yet), but it is technically required.
- `kwargs["code_system_url"]`: I didn't make this a required param (yet), but it is technically required.

## PyCharm signature method warning
I have a warning: 
> Signature of method 'OboGraphToFhirNpmConverter.dump()' does not match signature of the base method in class 'OboGraphToFhirJsonConverter'

It's unhappy that I made `target` a required param by removing ` = None`. But I don't see why this is invalid / bad. The code seems to run fine.

I can't seem to put a JetBrains `noinspection` comment to ignore the warning, either.

Reply:
@cmungall Basically the review I want here is that the nature of the FHIR output is now starting to deviate from what is standard in OAK. It's not just 1 file in, 1 file out for this. FHIR will now output a number of files, and if they're using the `fhirnpm` format, it'll be 1 zip.